quiz/generic/quiz_page.inc

```diff
@@ -344,10 +340,9 @@ function qp_echo_error_html($message_id)
     // Challenge the quiz-taker to fix the error.
     echo "<p>";
     if (isset($message["challengetext"])) {
+        // XXX: Currently unused. Might be discontinued.
```

Comment:
Should the "unused & might be discontinued" refer to the `if` or the `else`? It looks like the `else` from the original positioning of the comment, but the `if` with this edit. I'm sure you know which is correct - I don't, but it looked like a change.

Reply:
Good catch. My read is that it actually does belong with the `if`. `challengetext` is not used anywhere in the quizzes so I decided the comment must have applied to the `if`.

Reply:
Fair enough - thanks